cpp/recognition/CMakeLists.txt

```diff
@@ -5,3 +5,9 @@ koala_add_module(recognition
    perfect/Pyramids.cpp
    perfect/NearCleaners.cpp
 )
+koala_add_module(cograph
+   cograph/CorneilStewartPerlCographRecognition.cpp
+   CographRecognition.cpp
+   cograph/CoTree.cpp

```

Comment:
Please put CographRecognition.cpp as the first file

Reply:
fixed